Timer/ViewController.swift

```diff
@@ -1,19 +1,71 @@
-//
-//  ViewController.swift
-//  Timer
-//
-//  Created by Mark Wilhelm on 21.05.2022.
-//
-
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    var timer = Timer()
+    var isStarted: Bool = false
+    var isWorkTime: Bool = true
+    var secondsCurrent = 0
+    let secondsRest = 300
+    let secondsWork = 1500
+    
+    @IBOutlet weak var timeLabel: UILabel!
+    
+    var circularProgressBarView: CircularProgressBarView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        setUpCircularProgressBarView()
+    }
+    
+    func setUpCircularProgressBarView() {
+        circularProgressBarView = CircularProgressBarView(frame: .zero)
+        circularProgressBarView.center = view.center
+        view.addSubview(circularProgressBarView)
+    }
+    
+    @IBOutlet weak var startStopButton: UIButton!
+    @IBAction func startStopButton(_ sender: Any)
+    {
+        if (isStarted)
+        {
+            isStarted = false
+            timer.invalidate()
+            startStopButton.setTitle("Start", for: .normal)
+            
+        } else {
+            isStarted = true
+            startStopButton.setTitle("Stop", for: .normal)
+            timer = Timer.scheduledTimer(timeInterval: 1, target: self, selector: #selector(tick), userInfo: nil, repeats: true )
+            
+        }
     }
```

Comment:
Соблюдай правильное форматирование кода
```suggestion
    @IBAction func startStopButton(_ sender: Any) {
        if (isStarted) {
            isStarted = false
            timer.invalidate()
            startStopButton.setTitle("Start", for: .normal)
        } else {
            isStarted = true
            startStopButton.setTitle("Stop", for: .normal)
            timer = Timer.scheduledTimer(timeInterval: 1, target: self, selector: #selector(tick), userInfo: nil, repeats: true )
        }
    }
```